src/backend/gl/src/command.rs

```diff
@@ -67,8 +67,14 @@ pub enum Command {
         base_vertex: hal::VertexOffset,
         instances: Range<hal::InstanceCount>,
     },
+    DrawIndirect {
+        primitive: gl::types::GLenum,
+        offset: buffer::Offset,
+        draw_count: hal::DrawCount,
+    },
     BindIndexBuffer(gl::types::GLuint),
     //BindVertexBuffers(BufferSlice),
+    BindUniform(gl::types::GLuint, BufferSlice),
```

Comment:
We didn't do it so far but we should try to reflect the underyling gl function call name in these cmd names as it easily becomes quite confusing once the gl implementation becomes more sophisticated. Also difficult to reason about the required version.

Reply:
It's difficult when one command needs to do many things. It would get pretty ugly if we had one command for one operation... *shrug* let me know if you want anything split out